inspirehep/orcid/api.py

```diff
@@ -26,7 +26,7 @@ def _send_push_task(kwargs):
 def push_to_orcid(record):
     """If needed, queue the push of the new changes to ORCID."""
     if not current_app.config["FEATURE_FLAG_ENABLE_ORCID_PUSH"]:
-        logger.warning("ORCID push feature flag not enabled")
+        LOGGER.info("ORCID push feature flag not enabled")
```

Comment:
I would turn this into debug, wdyt?

Reply:
I don't think so, but what others think?